src/tests/SimTest.cpp

```diff
@@ -261,7 +336,9 @@ void SimTest::getDefaultLightingRGBAObservation() {
 void SimTest::getCustomLightingRGBAObservation() {
   Corrade::Utility::Debug()
       << "Starting Test : getCustomLightingRGBAObservation ";
```

Comment:
Why this? :)

(I'm aware that if you get a crash, it won't print what test case it started, but that shouldn't mean you have to suffer like this ;) On my list of things to fix.)